models/retip.js

```diff
@@ -16,5 +17,7 @@ module.exports = (sequelize, DataTypes) => {
     timestamps: true,
   });
   Retip.belongsTo(Tip(sequelize, DataTypes), { foreignKey: 'tipId' });
+  Retip.addHook('beforeCreate', async () => { await cache.del(['fetchStats']); });
+  Retip.addHook('beforeUpdate', async () => { await cache.del(['fetchStats']); });
```

Comment:
not after?

Reply:
does generally not make much of a difference since we are using the tips cache anyways but for readability it makes more sense